src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -43,9 +44,10 @@ public AddCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        Remark remark = new Remark("");
```

Comment:
If we are going to start out with an empty remark always, might be better to not change the Person constructor.

suggestion: 
1. if its always empty on instantiation, then don't require it as a parameter in the constructor OR
2. consider using method overloading for the constructor so it still works with previous versions

just suggestions, if you think this way is better then that's ok, but backwards compatibility is a really nice plus if you use method overloading

Reply:
Will consider this in v1.2b . 
will need to consider how i can add remark at the same time using add command. Currently trying out to standardise parseRemark but there are alot of components to change